src/xform_to_datalog/authorization_logic_test.h

```diff
@@ -0,0 +1,55 @@
+//-----------------------------------------------------------------------------
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//-----------------------------------------------------------------------------
+#include "src/xform_to_datalog/authorization_logic.h"
+
+#include <cstdlib>
+#include <filesystem>
+#include <fstream>
+#include <vector>
+
+#include "src/common/testing/gtest.h"
+
+namespace fs = std::filesystem;
+
+namespace raksha::xform_to_datalog {
+
+class AuthorizationLogicTest : public ::testing::Test {
+public:
+  static fs::path GetTestDataDir() {
+    const char* test_srcdir_env = std::getenv("TEST_SRCDIR");
+    const char* test_workspace_env = std::getenv("TEST_WORKSPACE");
+    EXPECT_NE(test_srcdir_env,  nullptr);
+    EXPECT_NE(test_workspace_env, nullptr);
```

Comment:
If you take my advice to convert these to `protected` fixture methods, these might work better as `CHECK`s. In a sense, that makes semantic sense: this isn't a predicate that you're testing about the program, but an assertion you're making about the test setup. If this continues to work as is, though, don't worry about this comment.

Reply:
Done.